lib/cable_ready/channel.rb

```diff
@@ -215,25 +29,9 @@ def add_operation(key, options)
     end
 
     def stub
-      {
-        add_css_class: [],
-        dispatch_event: [],
-        inner_html: [],
-        insert_adjacent_html: [],
-        insert_adjacent_text: [],
-        morph: [],
-        outer_html: [],
-        remove: [],
-        remove_attribute: [],
-        remove_css_class: [],
-        set_attribute: [],
-        set_cookie: [],
-        set_dataset_property: [],
-        set_style: [],
-        set_property: [],
-        set_value: [],
-        text_content: []
-      }
+      available_operations.each_with_object({}) do |available_operation, hash|
+        hash[available_operations] = []
```

Comment:
You have a typo here `available_operations` should be `available_operation`.

Reply:
lol nice catch